language/Korean/strings.po

```diff
@@ -2002,15 +2034,20 @@ msgid ""
 "If this is your first time using %2$s, please see the %2$s.pdf file in the "
 "docs folder for information on setting up and using %2$s."
 msgstr ""
+"시스템 ROM을 찾을 수 없습니다. 1$s.ini 파일에 지정된 롬 경로 설정을 확인하십"
+"시오.\n"
+"\n"
+"2$s를 처음 사용하는 경우 문서 폴더에 있는 %2$s.pdf 파일에서 %2$s 설정 및 사용"
+"에 대한 정보를 참조하세요."
```

Comment:
The folder in the MAME distribution is called “docs” which is not localised.  This is not referring to the documents folder from the user account profile.